MobileWallet/TariLib/Wrappers/Utils/MicroTari.swift

```diff
@@ -43,62 +43,91 @@ import Foundation
-        defaultFormatter.minimumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
-        defaultFormatter.maximumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
-        defaultFormatter.negativePrefix = "-"
-
-        withOperatorFormatter.numberStyle = .decimal
-        withOperatorFormatter.minimumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
-        withOperatorFormatter.maximumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
-        withOperatorFormatter.positivePrefix = "+ "
-        withOperatorFormatter.negativePrefix = "- "
-
-        preciseFormatter.numberStyle = .decimal
-        preciseFormatter.minimumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
-        preciseFormatter.maximumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
-        preciseFormatter.negativePrefix = "- "
-
-        preciseWithOperatorFormatter.numberStyle = .decimal
-        preciseWithOperatorFormatter.minimumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
-        preciseWithOperatorFormatter.maximumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
-        preciseWithOperatorFormatter.positivePrefix = "+ "
-        preciseWithOperatorFormatter.negativePrefix = "- "
+        MicroTari.defaultFormatter.numberStyle = .decimal
+        MicroTari.defaultFormatter.minimumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
+        MicroTari.defaultFormatter.maximumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
+        MicroTari.defaultFormatter.negativePrefix = "-"
+
+        MicroTari.withOperatorFormatter.numberStyle = .decimal
+        MicroTari.withOperatorFormatter.minimumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
+        MicroTari.withOperatorFormatter.maximumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
+        MicroTari.withOperatorFormatter.positivePrefix = "+ "
+        MicroTari.withOperatorFormatter.negativePrefix = "- "
+
+        MicroTari.preciseFormatter.numberStyle = .decimal
+        MicroTari.preciseFormatter.minimumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
+        MicroTari.preciseFormatter.maximumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
+        MicroTari.preciseFormatter.negativePrefix = "- "
+
+        MicroTari.preciseWithOperatorFormatter.numberStyle = .decimal
+        MicroTari.preciseWithOperatorFormatter.minimumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
+        MicroTari.preciseWithOperatorFormatter.maximumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
+        MicroTari.preciseWithOperatorFormatter.positivePrefix = "+ "
+        MicroTari.preciseWithOperatorFormatter.negativePrefix = "- "
+
+        MicroTari.editFormatter.numberStyle = .decimal
+        MicroTari.editFormatter.minimumFractionDigits = 0
+        MicroTari.editFormatter.maximumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
+        MicroTari.editFormatter.negativePrefix = "-"
+    }
+
+    public static func getTariConvertedNumber(_ number: NSNumber) -> UInt64 {
```

Comment:
Can we make this `getMicroTari` or something similar? 

Also have unit tests for this struct so just add to `testMicroTari()` any new functions

Reply:
This might not make sense, I'm just converting the number given as a parameter, not convereting the micro tari instance. Is that ok?